menu.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h2>My Recent Works</h2>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>Ruby On Rails</li>
```

Comment:
- [ ] Your work section currently has background images that are not in the Figma design, As we are meant to stick to the given design as much as we can in this milestone, I'll suggest you remove these background images. :smile: :+1: 

- SCREENSHOT FOR REFERENCE:
![Selection_037](https://user-images.githubusercontent.com/107632780/236638675-57a15d4b-c716-44fe-8346-9564b9b25222.png)